pkg/virt-handler/options.go

```diff
@@ -47,6 +47,34 @@ func virtualMachineOptions(
 	return options
 }
 
+func virtualMachineOptionsWithNetworkConfig(
```

Comment:
These wrappers are very noisy and do not scale well.

Would a direct approach like this work?
```go
option := VirtualMachineOptions(...)
option.SetIfaceDomainAttachment(netDomainAttachement(...)) 
```

Reply:
Ok, but that is a nit.
You can just do: `options.InterfaceDomainAttachment = netDomainAttachement(...)`, or do I miss something?

Reply:
Ahh I see now you did it already :)